nylas/client/client.py

```diff
@@ -145,8 +145,7 @@ def authentication_url(self, redirect_uri, login_hint=''):
                 'client_id': self.app_id,
                 'response_type': 'code',
                 'scope': 'email',
-                'login_hint': login_hint,
-                'state': generate_id()}
+                'login_hint': login_hint}
```

Comment:
We shouldn't remove the state param yet, so we could just pass an empty string for now until we add support for the argument in a future PR.